items/ARANYA_NPC.json

```diff
@@ -0,0 +1,16 @@
+{
+  "itemid": "minecraft:skull",
+  "displayname": "§9Aranya (NPC)",
+  "nbttag": "{SkullOwner:{Id:\"00000000-0000-0000-0000-000000000000\",Properties:{textures:[0:{Value:\"bc8ea1f51f253ff5142ca11ae45193a4ad8c3ab5e9c6eec8ba7a4fcb7bac40\"}]},Name:\"00000000-0000-0000-0000-000000000000\"},display:{Lore:[0:\"\"],Name:\"§9Aranya (NPC)\"},ExtraAttributes:{id:\"ARANYA_NPC\"}}",
```

Comment:
Why is the skull a steve head?

Reply:
Oh, it copied that way, might be because it's a spider, I'll check.